contributors/devel/api-conventions.md

```diff
@@ -156,6 +168,11 @@ API endpoints may deviate from the strict REST pattern and return resources that
 are not singular JSON objects, such as streams of JSON objects or unstructured
 text log data.
 
+Many simple kinds are used across all API groups and are thus considered part
```

Comment:
Such as what types? Where would I go to find them? Why would I care? Is it a correctness issue, or a code re-use best practice issue?